dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -332,38 +423,37 @@ public void write(OutputStream output) throws IOException, WebApplicationExcepti
     @GET
     @Path("/checkintegrity/{params:.*}")
     @Produces (MediaType.APPLICATION_JSON)
-    public Response checkIntegrity(@Context HttpServletRequest httpServletRequest, @Context final HttpServletResponse httpServletResponse, @PathParam("params") String params)  {
-        InitDataObject initData = webResource.init(params, httpServletRequest, httpServletResponse, true, null);
-
-        Map<String, String> paramsMap = initData.getParamsMap();
-
+    public Response checkIntegrity(@Context final HttpServletRequest httpServletRequest,
+                                   @Context final HttpServletResponse httpServletResponse,
+                                   @PathParam("params") final String params)  {
```

Comment:
Codacy found an issue: [The String literal "params" appears 6 times in this file; the first occurrence is on line 328](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)